7thWrapperLoader/dllmain.cpp

```diff
@@ -343,7 +343,7 @@ LONG WINAPI ExceptionHandler(EXCEPTION_POINTERS* ep)
     json data = json::parse(f);
 
     std::string version = data["runtimeOptions"]["framework"]["version"];
-    std::string msg = "Could not start the .NET Desktop Runtime version " + version + ".\nPlease make sure you have both the x86/x64 editions installed.\n\nVisit https://dotnet.microsoft.com for more information.";
+    std::string msg = "Could not start the .NET Desktop Runtime version " + version + ".\n Please make sure you have both the x86 and x64 editions installed.  You can also try\n\nusing the 7H exe installer or visit https://dotnet.microsoft.com for more information.";

```

Comment:
- `\n Please make sure...` -> `\nPlease make sure` ( remove the space )
- `You can also try\n\nusing the 7H exe installer...` -> `\n\nYou can also try using the 7H exe installer` ( remove the initial space and replace with `\n\n`, remove the other new lines )